src/pages-conditional/developers/local-environment.js

```diff
@@ -589,6 +601,14 @@ export const devtoolImage = graphql`
   }
 `
 
+export const devtoolLanguages = graphql`
```

Comment:
Can we wrap the entire GitHub query as a fragment, e.g. `repoInfo`?

```
export const repoInfo = graphql`
  fragment repoInfo on GitHub_Repository {
    stargazerCount
    languages(orderBy: { field: SIZE, direction: DESC }, first: 2) {
      nodes {
        name
      }
    }
    url
  }
`
```

That would reduce some of the duplication in our query.

Reply:
Thank you for this... this was the original goal but couldn't quite get it to work. This does! Just fixed.